doc/translations.md

```diff
@@ -3,78 +3,52 @@ Friendica translations
-If you want to help with translation for any language, be it correcting terms or translating friendica to a currently not supported language, please register an account at transifex.com and contact the friendica translation team there.
+The Friendica translation process is based on `gettext` PO files.
 
-Translating friendica is simple.
-Just use the online tool at transifex.
-If you don't want to deal with git & co. that is fine, we check the status of the translations regularly and import them into the source tree at github so that others can use them.
+Basic worflow:
+1. `xgettext` is used to collect translation strings across the project in the master PO file located in `view/lang/C/messages.po`.
+2. This file makes translations strings available at [the Transifex Friendica page](https://www.transifex.com/Friendica/friendica/dashboard/).
+3. The translation itself is done at Transifex by volunteers.
+4. The resulting PO files by languages are manually updated in `view/lang/<language>/messages.po`.
+5. PO files are converted to PHP arrays in `view/lang/<language>/strings.php` that are ultimately used by Friendica to display the translations.
 
-We do not include every translation from transifex in the source tree to avoid a scattered and disturbed overall experience.
-As an uneducated guess we have a lower limit of 50% translated strings before we include the language (for the core messages.po file, addont translation will be included once all strings of an addon are translated.
-This limit is judging only by the amount of translated strings under the assumption that the most prominent strings for the UI will be translated first by a translation team.
-If you feel your translation useable before this limit, please contact us and we will probably include your teams work in the source tree.
+## Translate Friendica in your favorite language
 
-If you want to help translating, please concentrate on the core messages.po file first.
-We will only include translations with a sufficient translated messages.po file.
-Translations of addons will only be included, when the core file is included as well.
+Thank you for your interest in improving Friendica's translation!
+Please register a free Transifex account and ask over at [the Transifex Friendica page](https://www.transifex.com/Friendica/friendica/dashboard/) to join the translation team for your favorite language.
 
-If you want to get your work into the source tree yourself, feel free to do so and contact us with and questions that arise.
-The process is simple and friendica ships with all the tools necessary.
+As a rule of thumb, we add support for a language in Friendica when at least 50% of the strings have been translated to avoid a scattered experience.
+For addons, we add support for a language when all the strings for this addon have been translated and only if we already support the language in Friendica.
```

Comment:
The first part of the condition is not true. We only include addon translation if the core translation is included. The condition also implies that addon translations that are not at 100% anymore (e.g. when the base strings change) are removed from the repository again.

In many cases the translation of the addons is done first, as they are smaller packages to translate, and the core translation is done later. They usually are at 100% then, but sometimes repeating texts like "Save settings" are only translated once but are part of every addon translation. So from that condition, those would never find their way into the repository then.

Reply:
I don't understand your comment, the first part of the condition isn't meant to be on its own without the `and only if we already support the language in Friendica`.

On top of that, it isn't a "hard" condition, just an explanation of the general process (hence the "rule of thumb"). You can still add (or remove support) for a language for an addon as you see fit.

Reply:
Simple

> For addons, we add support for a language when all the strings for this addon have been translated

This is not true. Addon translations are included if the core translation file is translated to more then 50%.